gce/prepull-1.18.yaml

```diff
@@ -0,0 +1,161 @@
+      # result:
+      #   grep -o -E 'e2eteam/([[:alnum:]_-]+):([[:alnum:]./_-]+)' \
+      #   build-log.txt | sort | uniq
+      # DaemonSets do not support a RestartPolicy other than 'Always', so we
+      # run ping in each container to keep it alive so that kubernetes does not
+      # continually restart the containers while we're prepulling.
+      containers:
+      - image: e2eteam/agnhost:2.8
+        name: agnhost-28
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/busybox:1.29
+        name: busybox1
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/cassandra:v13
+        name: cassandra5
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/curl:1803
+        name: curl2
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/dnsutils:1.1
+        name: dnsutils6
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/echoserver:2.2
+        name: echoserver7
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/entrypoint-tester:1.0
+        name: entrypoint-tester8
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/etcd:3.3.10
+        name: etcd9
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/etcd:3.4.3
+        name: etcd-343
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/etcd:v3.3.10
+        name: etcd10
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/fakegitserver:1.0
+        name: fakegitserver11
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/gb-frontend:v6
+        name: gb-frontend12
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/httpd:2.4.38-alpine
+        name: httpd-2438
+        resources:
+          requests:
+            cpu: 1m
+        command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
+      - image: e2eteam/httpd:2.4.39-alpine
```

Comment:
why are two versions of httpd needed?

Reply:
I don't know. This is just reflecting what's used by the tests right now.

Reply:
Yeah, there is a case for this, and the dup nginx. There are some deployment tests that upgrades the deployment images from one version to a newer one and checks that they're still working.